src/provider/snippetsProvider.ts

```diff
@@ -8,11 +8,12 @@ import { SnippetService } from '../service/snippetService';
 export class SnippetsProvider implements vscode.TreeDataProvider<Snippet>, vscode.TreeDragAndDropController<Snippet> {
     constructor(private _snippetService: SnippetService, private _extensionPath: string) { }
 
-    dropMimeTypes: readonly string[] = ['application/vnd.code.tree.snippetsProvider'];
+    dropMimeTypes: readonly string[] = ['application/vnd.code.tree.snippetsProvider', 'text/uri-list'];
```

Comment:
we won't need new mime type, we can use default one `application/vnd.code.tree.snippetsProvider`

Reply:
Done.